content/blog/online-privacy.de.md

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Most of the sections here look more or less exactly like the result Google Translate produces. We would rather not have machine-translated posts on our site. Not having a German translation is totally fine.